server/paths/manage.ts

```diff
@@ -98,6 +98,7 @@ const v2BedsPath = v2SinglePremisesPath.path('beds')
 const v2DateChangesPath = v2BookingPath.path('date-changes')
 const v2ExtensionsPath = v2BookingPath.path('extensions')
 const outOfServiceBedsPath = v2SinglePremisesPath.path('beds/:bedId/out-of-service-beds')
+const outOfServiceBedsIndexPath = path('/manage/out-of-service-beds')
```

Comment:
I think it would be worth creating a new path `const v2ManagePath = path('/manage')` and using it for this new path and `v2PremisesPath` - it will help if we ever need to change the path.

Reply:
good shout